src/Statement/StatementList.php

```diff
@@ -282,4 +282,24 @@ public function isEmpty() {
 		return empty( $this->statements );
 	}
 
+	/**
+	 * @since 3.0
+	 *
+	 * @param Statement $statement
+	 *
+	 * @return int|bool
+	 */
+	public function indexOf( Statement $statement ) {
+		$index = 0;
+
+		foreach ( $this->statements as $s ) {
+			if ( $s->equals( $statement ) ) {
```

Comment:
I'm aware of the difference. Is this an actual problem for the code using `Claims::indexOf`? Note how there can only be a single Claim with a specific GUID in Claims. While this is not the case with StatementList, in Wikibase Repo it generally does not make sense to end up with a StatementList that has Statements with the same GUID. Which is why I think this change does not matter.